pyproject.toml

```diff
@@ -54,6 +54,7 @@ dependencies = [
     'python-dateutil>=2.8,<3.0',
     'pytz>=2023.3,<2024.1',
     'beautifulsoup4[html5lib]>=4.11.1,<5.0',
+    'orjson>=3.10.15,<4.0'
```

Comment:
Do we have a PR to add this to `https://github.com/vertexproject/vtx-base-image` ?

Reply:
Incoming.